src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -440,6 +447,7 @@ def _get_install_info(cmd,
         "ACR_SYNC_TOKEN_PASSWORD": sync_password,
         "ACR_PARENT_GATEWAY_ENDPOINT": parent_gateway_endpoint,
         "ACR_PARENT_LOGIN_SERVER": parent_registry_endpoint,
-        "ACR_PARENT_PROTOCOL": "https"
+        "ACR_PARENT_PROTOCOL": "https",
+        "ACR_REGISTRY_CONNECTION_STRING": connection_string
```

Comment:
As we discussed, 
1. Only keep ACR_REGISTRY_CONNECTION_STRING and ACR_REGISTRY_LOGIN_SERVER
2. Remove warning
3. Update RP comment with breaking change info

https://github.com/Azure/azure-cli/blob/dev/.github/pull_request_template.md